test/rotation_tests.jl

```diff
@@ -178,6 +178,24 @@ all_types = (RotMatrix{3}, Quat, SPQuat, AngleAxis, RodriguesVec,
         end
     end
 
+    #########################################################################
+    # Test robustness of DCM to Quat function
+    #########################################################################
+    function nearest_orthonormal(not_orthonormal)
+      # Didn't see a function like this already in Rotations
```

Comment:
Probably can delete this comment - it's more of a PR meta-comment which doesn't add much to future maintainability of the package ;-)